about_us.html

```diff
@@ -72,16 +72,12 @@ <h1>ABOUT US</h1>
                         <h1>Mission Statement</h1>
 
                         <p>
-                            TrickFire is all about giving our members hands-on
```

Comment:
I think I might have confused you here.

TrickFire originally competed in the NASA Lunabotics RMC (Robotics Mining Competition) from 2016 to 2021. From 2021 to present time, we are competing in the URC (University Rover Challenge).

The mission statement should talk about how members get hands on education, real-world experience, collaboration, and so on. This section should quickly mention our RMC history as well as state that TrickFire is currently competing in the URC (which is different from the RMC!). 

You can keep the bullet points as is.